test/utils.js

```diff
@@ -62,6 +62,11 @@ const waitFor = async (timeDelay) => {
   }
 };
 
+async function deployContract(wallet, contractName, args = []) {
+  const factory = await ethers.getContractFactory(contractName, wallet);
+  return factory.deploy(...args);
```

Comment:
wait for confirmations (you can import network directly here), makes this live network compatible
https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/test/AxelarGateway.js#L80